zipline/api.py

```diff
@@ -53,5 +54,6 @@
     'execution',
     'math_utils',
     'slippage',
-    'time_rules'
+    'time_rules',
+    'calendars',
```

Comment:
I guess we never did discuss this piece of it. I had added it here initially for consistency with `date_rules` and `time_rules`, but I'm totally open to changing this to a more appropriate place.